app/server/routes/frontendCommon.ts

```diff
@@ -17,8 +14,14 @@ if (conf.ENVIRONMENT === Environments.PRODUCTION && !conf.CLIENT_DSN) {
 
 export const getRecaptchaPublicKey = async () => {
   try {
-    const recaptchaConfig = await recaptchaConfigPromise;
-    const recaptchaPublicKey = recaptchaConfig?.publicKey;
+    let recaptchaPublicKey: string | undefined;
+
+    if (conf.ENVIRONMENT === Environments.DEVELOPMENT) {
+      recaptchaPublicKey = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI";
```

Comment:
I've made it so test recaptcha is always used in development environment, assume this is ok?

Reply:
Assuming that is the public recaptcha development key:

https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do

👍